assets/controller.yaml

```diff
@@ -87,6 +87,7 @@ spec:
           image: ${RESIZER_IMAGE}
           args:
             - --csi-address=$(ADDRESS)
+            - --csiTimeout=120s
```

Comment:
Please run `make update` to generate the bindata

Reply:
Yep.